torch/onnx/utils.py

```diff
@@ -770,16 +781,14 @@ def _run_symbolic_function(g, n, inputs, env, operator_export_type=OperatorExpor
                 outputs = n.outputsSize()
                 attrs["outputs"] = outputs
                 return _graph_at(g, op_name, *inputs, aten=True, **attrs)
-
             else:
                 # Export it regularly
+                domain = ''
+                symbolic_fn = _find_symbolic_in_registry(domain, op_name, opset_version, operator_export_type)
+                if symbolic_fn is None:
+                    return None
```

Comment:
what happens for export type 'ONNX' when symbolic_fn is None?

Reply:
Symbolic cannot be None for ONNX export type. Because of line 742:

if operator_export_type == OperatorExportTypes.ONNX_FALLTHROUGH:
            # Use the original node directly
            return None

With ONNX export type, sym_registry.get_registered_op(op_name, domain, opset_version) will always be called,
which would through RuntimeError for unregistered ONNX ops.